collection-scripts/gather_core_dumps

```diff
@@ -4,10 +4,38 @@ CORE_DUMP_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_core_dumps"}
 
 mkdir -p ${CORE_DUMP_PATH}/
 
+function get_dump_off_node { 
+    #Start Debug pod force it to stay up for 60 seconds, supress Stdout 
+    oc debug node/$1 -- /bin/bash -c 'sleep 60' > /dev/null 2>&1 &
+
+    #Get name of Debug Pod for relevant node 
+    export DEBUG=$(oc get pods --all-namespaces | grep $1-debug | awk -F' ' '{print $2}')
+
+    # Make sure pod is avaliable
+    while [ -z "${DEBUG}" ]
```

Comment:
should we have a counter here as well to ensure this doesn't run forever?

Reply:
I agree